microlith/scripts/collect-information.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Couchbase, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file  except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the  License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Don't set euo pipefail - we want this script to be as resilient as possible
+set -x
+
+echo "Starting collect-information.sh..."
+
+tmpdir=${TEMPORARY_DIRECTORY:-$(mktemp -d)}
+exec &> >(tee -a "$tmpdir/collect-information.sh.log")
+
+cp /etc/couchbase-cluster-monitor-release.txt "$tmpdir"
```

Comment:
Should we just grab anything /etc/*-release.txt? Might be bit more future proof plus potentially gives us UBI info support too - they have /etc/redhat-release.txt I think.

Reply:
Ah good shout, will do

Reply:
Did you mean to add a wildcard here?

Reply:
I thought I did... must be going mad